tripal_chado/src/Plugin/Field/FieldType/ChadoOrganismTypeDefault.php

```diff
@@ -3,131 +3,259 @@
 namespace Drupal\tripal_chado\Plugin\Field\FieldType;
 
 use Drupal\tripal_chado\TripalField\ChadoFieldItemBase;
-use Drupal\tripal_chado\TripalStorage\ChadoVarCharStoragePropertyType;
 use Drupal\tripal_chado\TripalStorage\ChadoIntStoragePropertyType;
-use Drupal\tripal\TripalStorage\StoragePropertyValue;
-use Drupal\core\Form\FormStateInterface;
-use Drupal\core\Field\FieldDefinitionInterface;
+use Drupal\tripal_chado\TripalStorage\ChadoVarCharStoragePropertyType;
+use Drupal\tripal_chado\TripalStorage\ChadoTextStoragePropertyType;
 
 /**
- * Plugin implementation of Tripal organism field type.
+ * Plugin implementation of default Tripal organism field type.
  *
  * @FieldType(
  *   id = "chado_organism_type_default",
- *   label = @Translation("Chado Organism Reference"),
+ *   object_table = "organism",
+ *   label = @Translation("Chado Organism"),
  *   description = @Translation("A chado organism reference"),
  *   default_widget = "chado_organism_widget_default",
- *   default_formatter = "chado_organism_formatter_default"
+ *   default_formatter = "chado_organism_formatter_default",
  * )
  */
 class ChadoOrganismTypeDefault extends ChadoFieldItemBase {
 
-  public static $id = "chado_organism_type_default";
+  public static $id = 'chado_organism_type_default';
+  // The following needs to match the object_table annotation above
+  protected static $object_table = 'organism';
+  protected static $object_id = 'organism_id';
 
   /**
    * {@inheritdoc}
    */
   public static function mainPropertyName() {
-    return 'label';
+    // Overrides the default of 'value'
+    return 'species';
```

Comment:
We need to have a property that can provide the main value for this field and species isn't quite right.  In another comment I suggested we put back the `label` field but name it `scientific_name`.  Here we can use that property.

Reply:
This will be fixed by commit https://github.com/tripal/tripal/pull/1743/commits/720af13025158a14f6edc9d1a9f45d8458bcdaea